src/appshell/view/internal/splashscreen/newinstanceloadingscreenview.cpp

```diff
@@ -35,6 +38,12 @@ NewInstanceLoadingScreenView::NewInstanceLoadingScreenView(const QString& openin
 {
     setAttribute(Qt::WA_TranslucentBackground);
 
+    QGraphicsDropShadowEffect* shadow = new QGraphicsDropShadowEffect();
+    shadow->setBlurRadius(12);
+    shadow->setColor("#40000000");
+    shadow->setOffset(0, 8);
+    setGraphicsEffect(shadow);
```

Comment:
I'd wrap this in `#ifndef Q_OS_MAC`, because on macOS the system already provides a beautiful drop shadow.